Admin ui features - Mariann- ready to review.md (#72)
SenseNet/docs.sensenet.com: content/guides/versioning.md

---
title: "Versioning"
metaTitle: "admin ui - Versioning"
metaDescription: "admin ui Versioning"
---

> Verioning is the management of changes to documents and other information stores in a file system or respository. With versioning function you can keep the published version of a document visible to non-editor users, while you are working on the latest, draft version.

Reviewer: ```suggestion
Versioning is the management of changes to documents and other informations stored in a file system or respository. With versioning you can keep the published version of a document visible to non-editor users, while you are working on the latest, draft version.
```

> In sensenet versioning is disabled by default.

Reviewer: ```suggestion
> In sensenet versioning is disabled by default.
```

> # How to set versioning on the admin ui?

Reviewer: ```suggestion
# How to set versioning on the admin-ui?
```

You can enable versioning for folders or content lists by selecting the folder/file, clicking _Edit_ and selecting the _Versioning Mode_.

> ![versioning](/content/guides/img/versioning.png)

Reviewer: ```suggestion
![versioning](./img/versioning.png)
```

> Subfolders inherit versioning settings by default.

Reviewer: ```suggestion
> Subfolders inherit versioning settings by default.
```

# Versioning modes

> _None_: The default setting of the Root folder, no versioning.

Reviewer: ```suggestion
**_None_**: The default setting of the Root folder, no versioning.
```

> _Inherited_: The folder inherits its versioning mode from its parent. This is the default setting for all other content.

Reviewer: ```suggestion
**Inherited**: The folder inherits its versioning mode from its parent. This is the default setting for all other content.
```

> _Major only_: Only major versions (1.0, 2.0,…) are preserved. After a user finished editing a Content, the major version number is bumped by default: eg. 1.0 becomes 2.0. This means the new version is automatically published, and will be served to all users requesting the content in question.

Reviewer: ```suggestion
**Major only**: Only major versions (1.0, 2.0,…) are preserved. After a user finished editing a Content, the major version number is bumped by default: eg. 1.0 becomes 2.0. This means the new version is automatically published, and will be served to all users requesting the content in question.
```

> _Major only_: Only major versions (1.0, 2.0,…) are preserved. After a user finished editing a Content, the major version number is bumped by default: eg. 1.0 becomes 2.0. This means the new version is automatically published, and will be served to all users requesting the content in question.

Reviewer: ```suggestion
_Major only_: Only major versions (1.0, 2.0,…) are preserved. After a user finished editing a content, the major version number is bumped by default (e.g. 1.0 becomes 2.0). This means the new version is automatically published, and will be served to all users requesting the content in question.
```

> _Major and minor_: Every version is preserved (1.0, 1.1, 1.2,…). In this mode, saving a Content bumps its minor version number. Content with minor version (for example 1.2, 3.5, anything but x.0) are considered working versions, and aren’t served to guest users - they get the latest major version instead. (eg. if a document’s latest version is 5.4D, guest users will be served 5.0A, the latest public version.) In this versioning mode, bumping the major version and thus publishing the changes can be done by pressing the Publish button. You can change versioning or approving mode on any folder or other container type. If you visit the edit page of the folder you'll find the versioning and approving settings among the advanced fields.

Reviewer: ```suggestion
**Major and minor**: Every version is preserved (1.0, 1.1, 1.2,…). In this mode, saving a Content bumps its minor version number. Content with minor version (for example 1.2, 3.5, anything but x.0) are considered working versions, and aren’t served to guest users - they get the latest major version instead. (e.g. if a document’s latest version is 5.4D, guest users will be served 5.0A, the latest public version.) In this versioning mode, bumping the major version and thus publishing the changes can be done by pressing the Publish button. You can change versioning or approving mode on any folder or other container type. If you visit the edit page of the folder you'll find the versioning and approving settings among the advanced fields.
```

> Following example shows you how to checkout (lock) a document:

Reviewer: ```suggestion
The following example shows you how to checkout (lock) a document:
```

> ![checkout_file](/content/guides/img/checkout_file.png)

Reviewer: ```suggestion
![checkout_file](./img/checkout_file.png)
```

In the same way you can checkin the file.

To know if a file is currently checked out see the symbol.

> ![locked_file](/content/guides/img/locked_file.png)

Reviewer: ```suggestion
![locked_file](./img/locked_file.png)
```
